### PR TITLE
INT-7026 allow next operator on openshift 4.10+

### PR DESCRIPTION
**Pull request by jflinchbaugh**
ready for openshift 4.12 in december 2022: adjust version declaration

JIRA: https://issues.sonatype.org/browse/INT-7026
Build: N/A

### CODE REVIEW

File: scripts/templates/annotations.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@ annotations:
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: nxrm-operator-certified
-  com.redhat.openshift.versions: v4.6-v4.11
+  com.redhat.openshift.versions: v4.10
```
Comment (jflinchbaugh) [inline scripts/templates/annotations.yaml:9]: this format means "4.10 and later". old range is what you think it is.  locking it to "only v4.10" would be "=v4.10".

File: scripts/bundle.sh [hunk 2]
```diff
@@ -52,7 +52,7 @@ mv bundle.Dockerfile bundle-$latest_version.Dockerfile
 
 # append more labels
 cat >> bundle-$latest_version.Dockerfile <<EOF
-LABEL com.redhat.openshift.versions="v4.8"
+LABEL com.redhat.openshift.versions="v4.10"
```
Comment (jflinchbaugh) [inline scripts/bundle.sh:55]: this label has the same name as the annotation, so I updated it, but the docs don't mention it, so I'll not be surprised if it no longer exists in operators generated with new operator-sdk.

**Review by hectorhuol — APPROVED** [review]
+1

**Review by eduard-tita — APPROVED** [review]

**Review by amoreno-sonatype — APPROVED** [review]
+1

**Pull request merged**